mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go

```diff
@@ -1308,6 +1336,71 @@ func TestAccSQLDatabaseInstance_DenyMaintenancePeriod(t *testing.T) {
 	})
 }
 
+func TestAccSqlDatabaseInstance_EnterprisePlusEdition(t *testing.T) {
+	t.Parallel()
+	enterprisePlusName := "tf-test-enterprise-plus" + acctest.RandString(t, 10)
+	enterprisePlusTier := "db-perf-optimized-N-2"
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccSqlDatabaseInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
```

Comment:
Is it possible to combine this test case with TestAccSqlDatabaseInstance_EnterpriseEdition? After combination, it tests both EnterprisePlusEdition and EnterpriseEdition. It also tests the updating of the filed edition